README.md

```diff
@@ -17,6 +17,11 @@ AVD Documentation:
 - [Stable version](https://avd.arista.com/stable/)
 - [Development version](https://avd.arista.com/devel/)
 
+## Support
```

Comment:
this should be added to root index.md and collections README.md (with direct links)